gsk4/src/auto/render_node.rs

```diff
@@ -21,7 +21,7 @@ glib_wrapper! {
 }
 
 impl RenderNode {
-    //pub fn deserialize(bytes: &glib::Bytes, error_func: /*Unimplemented*/FnMut(/*Unimplemented*/Fundamental: Pointer, &glib::Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Option<RenderNode> {
+    //pub fn deserialize(bytes: &glib::Bytes, error_func: /*Unimplemented*/FnMut(/*Ignored*/gtk::CssSection, &glib::Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Option<RenderNode> {
```

Comment:
This causes a circular dependency between gtk4 and gsk4, that's rather suboptimal.